XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/MixedRealityControllerVisualizationProfile.cs

```diff
@@ -99,20 +118,21 @@ public GameObject GlobalRightHandModel
         /// </summary>
         /// <param name="controllerType">The type of controller to query for</param>
         /// <param name="hand">The specific hand assigned to the controller</param>
-        public GameObject GetControllerModelOverride(Type controllerType, Handedness hand)
+        public bool GetControllerModelOverride(Type controllerType, Handedness hand, out GameObject overrideModel)
```

Comment:
Minor:  missing param doc, maybe worth to mention that it will be set to null when no type was found